templates/index.html

```diff
@@ -14,200 +14,221 @@
-            <h3 class="p-matrix__title">Featureful</h3>
-            <div class="p-matrix__desc">
-              <p>
-                All the cool things you probably want to try on a small, standard OpenStack are all built-in.
-              </p>
-            </div>
-          </div>
-        </li>
-        <li class="p-matrix__item">
-          <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Small</h3>
-            <div class="p-matrix__desc">
-              <p>
-                Use MicroStack in your CI/CD pipelines and get on with your day without headaches.
-              </p>
-            </div>
-          </div>
-        </li>
-      </ul>
+      <div class="col-6 p-card">
+        <h3>OpenStack community project</h3>
+        <hr class="u-sv1">
+        <p>MicroStack is an Openstack community project with optional commercial support available from Canonical. It benefits from the 10 year maturity of OpenStack, its stability and community collaboration.</p>
+        <p><a href="https://discourse.charmhub.io/search?q=microstack" class="p-link--external">Join the community</a></p>
+      </div>
+      <div class="col-6 p-card">
+        <h3>All-in one OpenStack</h3>
+        <hr class="u-sv1">
+        <p>MicroStack is an OpenStack in a snap. This means that all OpenStack components and their dependencies are packaged together inside of a single image that runs in isolation, ensuring high-level security.</p>
+        <p><a href="https://snapcraft.io/microstack" class="p-link--external">Learn more about snaps</a></p>
+      </div>
+      <div class="col-6 p-card">
+        <h3>Single-node OpenStack</h3>
+        <hr class="u-sv1">
+        <p>Single-node installation is usually the first step to try OpenStack. MicroStack supports this use case, providing the ability to deploy fully functional OpenStack on a single machine with just 2 commands.</p>
+        <p><a href="/docs/single-node">Read single-node installation instructions&nbsp;&rsaquo;</a></p>
+      </div>
+      <div class="col-6 p-card">
+        <h3>Multi-node OpenStack</h3>
+        <hr class="u-sv1">
+        <p>In order to meet the demand for multi-node OpenStack deployments, MicroStack also supports clustering with one node acting as a cloud controller and other nodes acting as hypervisors.</p>
+        <p><a href="/docs/multi-node">Read multi-node installation instructions&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="col-7">
+        <h2>OpenStack on rails</h2>
+        <h4>OpenStack has tens of components and options. Not everyone needs this complexity.</h4>
+       </div>
+      <div class="col-6">
+        <p>MicroStack includes core OpenStack services and the most popular compute, network and storage options. This eliminates the OpenStack complexity and provides an “on rails” experience.</p>
+      </div>
+      <div class="col-8 u-equal-height">
```

Comment:
There are too many cols in this row - you can only have a maximum of 12 cols in every row. Can you close the row before the col-8? Then I don't think you need the col-8 you can just replace this with a row and then each of the list cols can be a col-3? Try that and let me know if you need help

Reply:
Fixed! :)